thunder/executors/pythonex.py

```diff
@@ -365,5 +365,11 @@ def _elementwise_binary_checker(a: NumberLike | TensorProxy, b: NumberLike | Ten
 ex.register_implementation(prims.sub, sub, checker=_elementwise_binary_checker)
 ex.register_implementation(prims.div, div, checker=_elementwise_binary_checker)
 
+def _sink(*args, **kwargs):
+    return
+
+sink = ex.register_operator("sink", like=prims.sink, fn=_sink)
+ex.register_implementation(prims.sink, sink, checker=_always_executable)
```

Comment:
I added a no-op to have pythonex claim the sink prim.

Alternatively we can just remove this node after prologue is constructed. I'm open to suggestions here.

Reply:
I think it is fine as is. Specifically, I would avoid removing early because with early transforms, it'll be funny...